### ISSUE TITLE
Disabling FutureWarnings on Python3.6

### ISSUE THREAD

**Issue by pawelkopec** [brian-team/brian2#942]
One idea from me for better learning experience. When running first exercises on Python3.6 Jupyter notebook:
https://github.com/EPFL-LCN/neuronaldynamics-exercises/blob/master/doc/exercises/leaky-integrate-and-fire.rst
I got annoying warnings that disturb readability:
```python
/usr/local/lib/python3.6/dist-packages/brian2/core/variables.py:174: FutureWarning: Conversion of the second argument of issubdtype from `float` to `np.floating` is deprecated. In future, it will be treated as `np.float64 == np.dtype(float).type`.
  return np.issubdtype(np.bool, self.dtype)
WARNING    /usr/local/lib/python3.6/dist-packages/brian2/core/variables.py:174: FutureWarning: Conversion of the second argument of issubdtype from `bool` to `np.generic` is deprecated. In future, it will be treated as `np.bool_ == np.dtype(bool).type`.
  return np.issubdtype(np.bool, self.dtype)
 [py.warnings]
WARNING    /usr/local/lib/python3.6/dist-packages/brian2/monitors/statemonitor.py:287: FutureWarning: Conversion of the second argument of issubdtype from `int` to `np.signedinteger` is deprecated. In future, it will be treated as `np.int64 == np.dtype(int).type`.
  if np.issubdtype(dtype, np.int):
 [py.warnings]
WARNING    /usr/local/lib/python3.6/dist-packages/brian2/monitors/statemonitor.py:57: FutureWarning: Conversion of the second argument of issubdtype from `int` to `np.signedinteger` is deprecated. In future, it will be treated as `np.int64 == np.dtype(int).type`.
  if np.issubdtype(dtype, np.int) and not isinstance(item, np.ndarray):
 [py.warnings]
```
You can add this before your imports to disable those logs:
```python
import warnings
warnings.simplefilter(action='ignore', category=FutureWarning)
```

**Comment by mstimberg** [comment]
Thanks for the comment. This will no longer be necessary with the upcoming release (or if you use Brian2's version from master), as we've already fixed the statements responsible for the warnings (see #913).

**Comment by mstimberg** [comment]
This should be fix with the latest release (currently: 2.1.3.1), please re-open this issue if you still see such warnings.
